libs/utils/env.py

```diff
@@ -644,6 +644,17 @@ def _init_platform(self):
         # Adding topology information
         self.platform['topology'] = self.topology.get_level("cluster")
 
+        # Adding kernel build information
+        kver = self.target.execute('uname -r').strip()
```

Comment:
Added this new interface in:
   https://github.com/ARM-software/devlib/pull/90

... let's wait for this to be merged and than I'll add a refresh of devlib to this PR.